### ISSUE TITLE
Additional data required in package.json for dependencies

### ISSUE THREAD

**Issue by jsobell** [grofit/treacherous-aurelia#1]
When webpack include the module it fails to include any of the associates behaviors, attributes or elements.
The following section is needed in the package.json:

```
  "aurelia": {
    "build": {
      "resources": [
        "binding-behaviours/validate-binding-behaviour",
        "attributes/validation-group-attribute",
        "attributes/validation-options-attribute",
        "attributes/validate-property-attribute",
        "elements/validation-summary-element",
        "elements/validation-summary.html"
      ]
    }
  },
```

I assume this is because the only reference to those dependencies is `aurelia.globalResources("./binding-behaviours/validate-binding-behaviour");` etc, so the package generator has no reference to the classes to include in the resulting bundle.

It also still requires

```
  "dependencies": {"treacherous":"^0.9.0"},
```

**Comment by jsobell** [comment]
Resolved in PR #2

**Comment by grofit** [comment]
merged your change, should fix this.
